html/net.js

```diff
@@ -107,6 +107,9 @@ export class ServerConnection extends ServerConnectionBase {
       n_samples: chunk.length,
       ... this.send_metadata
     });
+    if (!response) {
+      return null;
```

Comment:
Everywhere else if we fail we return something that looks like `{ ... chunk: null }` rather than just `null`. We should probably be consistent. And I think in `handle_message` you check for response.chunk == null, not for response == null. So you should probably make this match the other cases.

Reply:
We check for both, in order, and they do different things: if `response === null`, which happens if the XHR fails (usually due to a connectivity problem), we `await try_increase_batch_size_and_reload()` before returning (https://github.com/gwillen/solstice-audio-test/blob/77de658a5f7aff31088fd41a6acb978644e6f2b0/html/app.js#L1196). We don't do that if `response.chunk === null`.

Reply:
aha okay, thanks, looks good then.